tox.ini

```diff
@@ -33,7 +33,8 @@ commands =
     # Clean up from previous tests
     python {toxinidir}/util.py clean-tests
     # Run tests
-    pytest -vv -rsfE --cov=sqlfluff --cov-report=xml --cov-report=lcov {posargs: {toxinidir}/test} -m "not integration_test"
+    pytest -vv -rsfE --cov-report=xml --cov-report=lcov {posargs: {toxinidir}/test} -m "not integration_test"
+    python test/patch_lcov.py
```

Comment:
I don't think if I understand *why* we need to do this 🤔 . I don't think it's harmful, so no need to disapprove, but if you're able to add some commentary here (and/or in the `.py` file) then I think that makes this more future proof.

Reply:
Will do.

I'm adding this to the docstring in the `patch_lcov.py` script.

```
Context: When the CI build runs tests, it uses tox, which installs SQLFluff
in a virtual environment. Thus, iThe lcov file generated by the tests contains
paths to the virtual environment. This script replaces those paths with
paths relative to the repo root. This allows the lcov file to be used by
Coveralls. Without this, Coveralls has valid coverage info, but it generates
URLs that point to source files that don't exist in the SQLFluff GitHub repo.

For example, we want to change this:
SF:.tox/py/lib/python3.10/site-packages/sqlfluff/__init__.py

to this:
SF:src/sqlfluff/__init__.py
```